gitlab/pipeline.ml

```diff
@@ -4,6 +4,18 @@ module Git = Current_git
 module Gitlab = Current_gitlab
 module Docker = Current_docker.Default
 
+module Metrics = struct
+  open Prometheus
+
+  let namespace = "ocurrent"
```

Comment:
```suggestion
  let namespace = "ocaml-ci"
```

Reply:
This aligns with the equivalent metrics in the GitHub plugin https://github.com/ocurrent/ocurrent/blob/master/plugins/github/installation.ml#L4-L13

In future the GitLab plugin will have a table of registered org/repos that can feed this Metric.

Reply:
Oh, is it because this is considered to be `current_gitlab`? :thinking:

Reply:
Basically yes.